apps/website/src/components/chart/chart.tsx

```diff
@@ -0,0 +1,48 @@
+import { Stack } from '@tamagui/core'
+import { ParentSize } from '@visx/responsive'
+
+import { ChartComponent, Empty, Loading } from './components'
+
+type DayType = {
+  date: string
+  open_issues: number
+  closed_issues: number
+}
+
+interface Props {
+  data: DayType[]
+  width?: number
```

Comment:
Let's optimize for the current use case and not expose more props than necessary and remove `width`.